library/core/src/panicking.rs

```diff
@@ -212,6 +212,7 @@ panic_const! {
 
 /// Like `panic`, but without unwinding and track_caller to reduce the impact on codesize on the caller.
```

Comment:
Based on the function documentation it appears to be intentionally not using `track_caller`.

Reply:
Yeah, I was involved in the discussion on the PR that added this comment in the first place :)
https://github.com/rust-lang/rust/pull/102732

I created this PR in part to evaluate whether the code size concerns are ill-founded. I suspect they are.